pkgs/tools/audio/beets/common.nix

```
@@ -144,6 +145,7 @@ python3Packages.buildPythonApplication {
       mock
       rarfile
       responses
+      writableTmpDirAsHomeHook
     ]
```

Personally I don't mind that much the fact a non Python package is mixed here, but I was thinking of:

```suggestion
    ] ++ [
      writeableTmpDirAsHomeHook
    ]